po/nl.po

```diff
@@ -325,23 +323,20 @@ msgid "No software manager found"
 msgstr "Geen softwarebeheerder aangetroffen."
 
 #: src/widgets/menu.ui:7
-#, fuzzy
 msgid "_Help"
-msgstr "Hulp"
+msgstr "_Hulp"
 
 #: src/widgets/menu.ui:11
-#, fuzzy
 msgid "_Documentation"
-msgstr "Documentatie"
+msgstr "_Documentatie"
 
 #: src/widgets/menu.ui:15
 msgid "_Keyboard Shortcuts"
-msgstr ""
+msgstr "_Sneltoetsen"
```

Comment:
I think you're already marking an `s` in `Detail_s tonen`, so I think it would be better to mark another character here, e.g. `S_neltoetsen` ?

Reply:
You're right, my bad - should've catched that one! Anyway, I've corrected it just now, thanks! 🙂